ext/readline/readline.c

```diff
@@ -307,6 +315,23 @@ PHP_FUNCTION(readline_info)
 			RETVAL_STRING(SAFE_STRING(rl_prompt));
 		} else if (!strcasecmp(what, "terminal_name")) {
 			RETVAL_STRING((char *)SAFE_STRING(rl_terminal_name));
+		} else if (!strcasecmp(what, "completion_suppress_append")) {
+			oldval = rl_completion_suppress_append;
+			if (value) {
+				convert_to_long_ex(value);
+				rl_completion_suppress_append = Z_LVAL_P(value);
```

Comment:
For consistency this should use `convert_to_boolean_ex`. Or more simply just `rl_completion_suppress_append = zend_is_true(value)`.

Reply:
Ah I see, that does make more sense.